pkg/controller/registration/registration_controller.go

```diff
@@ -247,6 +221,14 @@ func registerDefaultController(r *ReconcileRegistration, config *controller.Conf
 		}
 
 	default:
+		// register controllers for direct CRDs
+		if val, ok := crd.Labels[k8s.DirectCRDLabel]; ok && val == "true" {
```

Comment:
Might be easier to have a `directbase.ControllerBuilder.IsDirect` helper, than to require the additional label.

Reply:
I don't have a preference. This is just to keep the same code style as below (line 232 and line 241).

Reply:
so as the function name "Add" vs "AddController"